html/semantics/forms/the-fieldset-element/accessibility/README.md

```diff
@@ -0,0 +1,5 @@
+Fieldset accessibility tests
+============================
+
+These tests are intended to test the accessibility of the fieldset and legend elements. They do not
+yet have pass conditions or a clear way to run them. This is expected to be addressed later.
```

Comment:
This makes them rather hard to review, no?

Reply:
Yeah. :/ I don't know yet what expected behavior should be.

Reply:
Also see https://github.com/w3c/html-aam/issues/145